### ISSUE TITLE
darwin/amd64 build fails on wails init basic repo

### ISSUE THREAD

**Issue by lradomirov** [wailsapp/wails#815]
**Description**

Creating a new wails repository with `wails init`, then running `wails build -x darwin/amd64` results in a build failure.

**To Reproduce**
Steps to reproduce the behaviour:
1. initialize a new repository with `wails init`.
2. run `wails build -x darwin/amd64` within the newly created repository.

**Expected behaviour**
I would expect a macOS binary to be in the build directory.  See logs below in build.txt.

[build.txt](https://github.com/wailsapp/wails/files/7192661/build.txt)

**System Details**

| Name   | Value |
| ----- | ----- |
| Wails Version | v1.16.7 |
| Go Version    | go1.16.7 |
| Platform      | linux |
| Arch          | amd64 |
| GO111MODULE   | (Not Set) |
| GCC           | 9.3.0 |
| Npm           | 7.20.2 |
| Node          | v14.17.3 |

- [ ] This issue is for Windows and I have read the [Windows Developer Guide](https://wails.app/guides/windows/)

**Comment by lradomirov** [comment]
https://github.com/wailsapp/wails/issues/649#issuecomment-823152948 seems to be related.

**Comment by leaanthony** [comment]
Cross compilation broke due to moving to Go 1.16 breaking upstream xgo. That was fixed reltively recently so we are in the process of moving to that.

**Comment by lradomirov** [comment]
@leaanthony https://github.com/crazy-max/xgo/issues/28, recent indeed!  Thank you.  

I got `darwin/amd64` to build successfully by downloading the latest release from https://github.com/crazy-max/xgo and running `path/to/crazy-max/xgo --targets=darwin/amd64 ./` in my `wails init` repo directory.  I tested the binary on Mac 11 and it ran without errors.

I'll leave this issue open until you finalize the changes for anyone else encountering this issue.
